test/framework/src/main/java/org/opensearch/test/OpenSearchIntegTestCase.java

```diff
@@ -1904,7 +1904,9 @@ protected Settings nodeSettings(int nodeOrdinal) {
             .put(SearchService.LOW_LEVEL_CANCELLATION_SETTING.getKey(), randomBoolean())
             .putList(DISCOVERY_SEED_HOSTS_SETTING.getKey()) // empty list disables a port scan for other nodes
             .putList(DISCOVERY_SEED_PROVIDERS_SETTING.getKey(), "file")
-            .put(featureFlagSettings());
+            .put(featureFlagSettings())
+            .put(TelemetrySettings.TRACER_ENABLED_SETTING.getKey(), true)
+            .put(FeatureFlags.TELEMETRY_SETTING.getKey(), true);
```

Comment:
I still don't understand why `FeatureFlags.TELEMETRY_SETTING.getKey(), true)` is added in node settings - we have `featureFlagSettings()` for that, please move it there

Reply:
We want to add telemetry validations by default for all integ test cases to avoid memory leaks. I see we have included this in SingleNodeTestCase https://github.com/opensearch-project/OpenSearch/blob/e1a41255de203bc57c1e29469a9bb5d50fe0d84b/test/framework/src/main/java/org/opensearch/test/OpenSearchSingleNodeTestCase.java#L248. Same idea here to enable it by default for all integ test cases.

Reply:
Please leave the option to opt-out from this plugin, there are many plugins that will never use open telemetry, please follow the convention we have for plugins / feature flags / mock plugins ...

Reply:
Added in featureFlagSettings()